packages/lit-dev-content/site/docs/v3/components/decorators.md

```diff
@@ -77,6 +83,45 @@ import {customElement} from 'lit/decorators/custom-element.js';
 import {eventOptions} from 'lit/decorators/event-options.js';
 ```
 
+## Standard decorators { #standard-decorators }
+
+"Standard decorators" is the version of decorators that has reached Stage 3 consensus at TC39, the body that defines ECMAScript/JavaScript. The decorators specification has been fine-tuned with active VM implementor feedback to be as optimizable as possible.
+
+Standard decorators are supported in TypeScript and Babel, with native browser coming in the near future.
```

Comment:
Maybe less specific: "in the near future" -> "in the future".

Reply:
Near is vague enough, and I kind of like that this is optimistic! This sentence is more accurate now than it was 2 weeks ago, and will become more correct as time goes haha.

Reply:
haha! Agree!